.github/workflows/roman_ci.yml

```
@@ -91,8 +43,9 @@ jobs:
         DD_GIT_REPOSITORY_URL: ${{ github.repositoryUrl }}
         DD_GIT_COMMIT_SHA: ${{ github.sha }}
         DD_GIT_BRANCH: ${{ github.ref_name }}
-      cache-path: ${{ needs.data.outputs.data_path }}
-      cache-key: data-${{ needs.data.outputs.data_hash }}-${{ needs.data.outputs.crds_context }}
+      cache-path: ${{ needs.data.outputs.crds_path }}
```

shouldn't this be the generic data path?
```suggestion
      cache-path: ${{ needs.data.outputs.data_path }}
```

---

I ended up removing `data_path` as it was the same as `crds_path` (both were `/tmp/data`) and the jobs were inconsistent with usage (`roman_ci` used `data_path`, `roman_ci_cron` used `crds_path` for the cache path).

What do you think about re-introducing `data_path` as something like `/tmp/data` and have `crds_path` be `/tmp/data/crds` (now in a `crds` subdirectory) and `webbpsf_path` be `/tmp/data/webbpsf` (same as before)? That way if more data is needed it can go into a separate subdirectory.

---

yes that sounds good to me; also, we probably shouldn't be using the CRDS context in the cache key anyway, as CRDS files are only downloaded during tests not before; any downloaded CRDS files would not be saved